schedule/models/events.py

```diff
@@ -43,6 +44,8 @@ class Event(models.Model):
     calendar = models.ForeignKey(Calendar, null=True, blank=True, verbose_name=_("calendar"))
     objects = EventManager()
 
+    recent_start = models.TextField(null=True)
```

Comment:
If you do need to represent an object, might be worth going for a `JSONField` here so that you dont need to worry about pickling/unpickling and so that the values are easier to debug at a glance after making a query against the event table.